### DISCUSSION TITLE
Admin Approval

### DISCUSSION THREAD

**Discussion by davedave1234** [lithnet/access-manager#124] [Ideas]
I would like to know if there is a current way to, or something you can implement that would, allow me the ability to approve requests made by regular users.

For example: I want to give all my service desk access to request LAPS passwords all the time and just receive an email notifying that they did, this is easy and working great. I would also like to give my users ability to request LAPS passwords and/or JIT access but I would like an email sent with an "Approve" link, or something similar, to me and my administrators so that the users don't get access unless we approve and can not give them selves access without us being aware.

As a work around I have setup an authorization policy and as a procedural process it is enabled with an expiration time when a user needs access. This is a manual process and is open to user error.

I want to add that I love the product.

**Comment by brolifen** [comment]
Any answer on this if we're going to see a formal approval granting flow to requests?

**Reply by ryannewington** [reply]
It is on the back log of features under consideration, but it is not scheduled at this point.
